### DISCUSSION TITLE
EFI Vars

### DISCUSSION THREAD

**Discussion by sambow23** [sambow23/Dell-Precision-5540-macOS#6] [General]
#### Use with `modGRUBShell` tool In OpenCore (built into this EFI)
|  Var | Offset |
| ------------- | ------------- |
| setup_var_cv | Setup 0x6ED 0x01 0x00 |
| setup_var | 0xA10 0x02 |
| setup_var | 0xA11 0x03 |
| setup_var_3 | 0x789 0x00 |

Tested BIOS's
`1.31.0`, `1.39.0`

**Comment by licencedtokill** [comment]
Have you tried booting with Dell Bios 1.41.0? I can't get the Mac OS 15 recovery to load.

**Reply by sambow23** [reply]
Nope, though windows installed 1.39.0 on my machine at some point and that fixed most of the issues I had in macOS, you should be able to downgrade.
